docs/project/api-review-process.md

```diff
@@ -38,6 +38,16 @@ APIs and some code samples that show how it should be used. If changes are neces
     replace the label `api-ready-for-review` with `api-needs-work`.
     * **Rejected**. In case we believe the proposal isn't a direction we want to go after, we simply write a comment and close the issue.
 
+## Review schedule
+
+ There are three methods to get an API review:
+
+* **Get into the backlog**. Generally speaking, filing an issue in `dotnet/runtime` and slapping the label `api-ready-for-review` on it will make your issue show up during API reviews. The downside is that we generally walk the backlog oldest-newest, so your issue might not be looked at for a while.
+* **Fast track**. If you need to bypass the backlog apply both `api-ready-for-review` and `blocking`. All blocking issues are looked at before we walk the backlog.
```

Comment:
did you mention that `blocking` was confused (eg with `blocked`) ? Would it reduce, or increase confusion if we renamed it `blocking-other-work` or similar?

@karelz